[2024-10-11T14:59:11.034+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_sftp_s3_test.task_0_SFTP_to_S3 manual__2024-10-11T14:59:08.352867+00:00 [queued]>
[2024-10-11T14:59:11.043+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_sftp_s3_test.task_0_SFTP_to_S3 manual__2024-10-11T14:59:08.352867+00:00 [queued]>
[2024-10-11T14:59:11.044+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-10-11T14:59:11.045+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2024-10-11T14:59:11.047+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-10-11T14:59:11.068+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): task_0_SFTP_to_S3> on 2024-10-11 14:59:08.352867+00:00
[2024-10-11T14:59:11.090+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'pipeline_sftp_s3_test', 'task_0_SFTP_to_S3', 'manual__2024-10-11T14:59:08.352867+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_sftp_s3_test.py', '--cfg-path', '/tmp/tmp18scqnli']
[2024-10-11T14:59:11.090+0000] {standard_task_runner.py:55} INFO - Started process 32767 to run task
[2024-10-11T14:59:11.091+0000] {standard_task_runner.py:83} INFO - Job 4: Subtask task_0_SFTP_to_S3
[2024-10-11T14:59:11.121+0000] {logging_mixin.py:137} WARNING - /usr/local/lib/python3.9/site-packages/***/settings.py:250 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-11T14:59:11.271+0000] {task_command.py:389} INFO - Running <TaskInstance: pipeline_sftp_s3_test.task_0_SFTP_to_S3 manual__2024-10-11T14:59:08.352867+00:00 [running]> on host 9b96fcb811c3
[2024-10-11T14:59:11.436+0000] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pipeline_sftp_s3_test
AIRFLOW_CTX_TASK_ID=task_0_SFTP_to_S3
AIRFLOW_CTX_EXECUTION_DATE=2024-10-11T14:59:08.352867+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-10-11T14:59:08.352867+00:00
[2024-10-11T14:59:11.445+0000] {sftp_s3.py:158} INFO - Entered sftp_get_file function .....
[2024-10-11T14:59:11.448+0000] {logging_mixin.py:137} WARNING - /usr/local/lib/python3.9/site-packages/pysftp/__init__.py:61 UserWarning: Failed to load HostKeys from /root/.ssh/known_hosts.  You will need to explicitly load HostKeys (cnopts.hostkeys.load(filename)) or disableHostKey checking (cnopts.hostkeys = None).
[2024-10-11T14:59:11.449+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-10-11T14:59:11.453+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=pipeline_sftp_s3_test, task_id=task_0_SFTP_to_S3, execution_date=20241011T145908, start_date=20241011T145911, end_date=20241011T145911
[2024-10-11T14:59:11.481+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-10-11T14:59:11.536+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
